src-electron/db/query-package.js

```diff
@@ -896,6 +915,7 @@ exports.getPackageSessionPackagePairBySessionId =
   getPackageSessionPackagePairBySessionId
 
 exports.getPathCrc = getPathCrc
+exports.getZclPropertiesPackage = getZclPropertiesPackage
```

Comment:
@SaharKargar
Can you rebase your changes?
There are a few changes that is not related to data-* work.

Reply:
@jingteng25742 
Rebased and pushed.